samples/matter/template/README.rst

```diff
@@ -126,88 +126,102 @@ For example:
 
 .. matter_template_build_with_tfm_end
 
+Advanced configuration options
+==============================
+
+This section describes other configuration options for the sample.
+
 Device Firmware Upgrade support
-===============================
+-------------------------------
 
 .. |Bluetooth| replace:: Bluetooth
 
-.. include:: ../lock/README.rst
-    :start-after: matter_door_lock_sample_build_with_dfu_start
-    :end-before: matter_door_lock_sample_build_with_dfu_end
+.. toggle::
 
-Alternatively, for the nRF54L15 DK, the DFU can be configured to only use the internal MRAM for storage.
-This means that both the currently running firmware and the new firmware to be updated will be stored within the device's internal flash memory.
-This configuration is enabled by default for the :ref:`debug configuration <matter_template_custom_configs>`.
+   .. include:: ../lock/README.rst
+       :start-after: matter_door_lock_sample_build_with_dfu_start
+       :end-before: matter_door_lock_sample_build_with_dfu_end
 
-The following is an example command to build the sample on the nRF54L15 DK with support for Matter OTA DFU and DFU over Bluetooth SMP, and using internal MRAM only:
+   Alternatively, for the nRF54L15 DK, the DFU can be configured to only use the internal MRAM for storage.
+   This means that both the currently running firmware and the new firmware to be updated will be stored within the device's internal flash memory.
+   This configuration is enabled by default for the :ref:`debug configuration <matter_template_custom_configs>`.
 
-.. code-block:: console
+   The following is an example command to build the sample on the nRF54L15 DK with support for Matter OTA DFU and DFU over Bluetooth SMP, and using internal MRAM only:
 
-    west build -p -b nrf54l15dk/nrf54l15/cpuapp -- -DCONFIG_CHIP_DFU_OVER_BT_SMP=y -DFILE_SUFFIX=internal
+   .. code-block:: console
 
-To build the sample for the same purpose, but in the ``release`` configuration, use the following command:
+       west build -p -b nrf54l15dk/nrf54l15/cpuapp -- -DCONFIG_CHIP_DFU_OVER_BT_SMP=y -DFILE_SUFFIX=internal
 
-.. code-block:: console
+   To build the sample for the same purpose, but in the ``release`` configuration, use the following command:
+
+   .. code-block:: console
 
-    west build -p -b nrf54l15dk/nrf54l15/cpuapp -- -DCONFIG_CHIP_DFU_OVER_BT_SMP=y -DFILE_SUFFIX=internal -Dtemplate_EXTRA_CONF_FILE=prj_release.conf
+       west build -p -b nrf54l15dk/nrf54l15/cpuapp -- -DCONFIG_CHIP_DFU_OVER_BT_SMP=y -DFILE_SUFFIX=internal -Dtemplate_EXTRA_CONF_FILE=prj_release.conf
 
-Note that in this case, the size of the application partition is half of what it would be when using a configuration with external flash memory support.
+   .. note::
+      In this case, the size of the application partition is half of what it would be when using a configuration with external flash memory support.
 
 SUIT DFU on nRF54H20
```

Comment:
This has been just removed (with nRF54H20 support - so you may need to do rebase).